PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+
+## What is the type of this PR 
+
+e.g. Bug Fix, Feature, Code Quality Improvement, UI Polish...
+
+## Short description on what this PR suppose to do?
+
+e.g. Adding genuine check to the onboarding
+
+## Any special conditions required for testing?
+
+e.g. Clear db, add special env variable..
+
+## Any background context you want to provide?
+
+e.g. @gre worked on this previously, another requirement was added
+
+## What are the relevant tickets/issues?
```

Comment:
too much question. I think rephrase the previous question and merge the two. relevant issues are actually part of the context.

also, probably it's better to put the Context as one of the first question.